source/python/libpytimemory.cpp

```diff
@@ -92,6 +93,63 @@ struct pyenumeration
     }
 };
 
+struct pycomponents
```

Comment:
Remove `pycomponents` and do a `git pull upstream timem-mpi-and-pycomponents` (assuming `git remote add upstream https://github.com/NERSC/timemory.git`) so that you pull in the more extensive python bindings that I wrote. Keep the trace bindings though